jobs/start-devappserver-vm.groovy

```diff
@@ -0,0 +1,123 @@
+// Start up a VM for a developer use to run dev-appserver on.
+//
+// Developers can run a dev-appserver -- a local version of our
+// production website -- on their local machines, but it's slow and
+// uses a lot of resources.  To help with that, we provide a VM for this
+// purpose as well.  Users can copy their webapp dir over to this VM
+// and then ssh to it and run `make serve`.
```

Comment:
Ok so my mental flow was the inverted.  Now it makes sense why you were most likely scratching your head when I suggested sshfd.  I was thinking that the VM would be the one checking out the code and creating a branch for you.